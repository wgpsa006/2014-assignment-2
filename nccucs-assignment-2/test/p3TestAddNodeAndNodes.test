# Create graphs A and B, nodes n1 and n2
CreateGraph A
CreateGraph B
CreateNode n1 3
CreateNode n2 6
# Test empty lists of nodes
ListNodes A
ListNodes B
AddNode A n1
# Test that A has n1 and B has no nodes
ListNodes A
ListNodes B
AddNode B n1
# Test that A and B have n1
ListNodes A
ListNodes B
AddNode A n2
# Test that A has n2
ListNodes A