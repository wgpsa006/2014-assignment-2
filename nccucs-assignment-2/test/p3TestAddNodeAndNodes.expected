# Create graphs A and B, nodes n1 and n2
created graph A
created graph B
created node n1 with cost 3
created node n2 with cost 6
# Test empty lists of nodes
A contains:
B contains:
added node n1 to A
# Test that A has n1 and B has no nodes
A contains: n1
B contains:
added node n1 to B
# Test that A and B have n1
A contains: n1
B contains: n1
added node n2 to A
# Test that A has n2
A contains: n1 n2